examples/bpi-monolith-test-suite/bpi/components/remotes/internal.axios.ts

```diff
@@ -0,0 +1,7 @@
+import Axios from "axios";
+
+const server = "http://localhost:9000";
```

Comment:
env var would be good for this, check dotenv package for example

Reply:
This file removed as unused code